test/test-cases/export-class/expected.css

```diff
@@ -19,5 +23,6 @@
 
 :export {
   exportName: _input__exportName;
+  ::::: _input__\:\:\:\:;
```

Comment:
`::::` still invalid and can't be parsed `postcss`, we should fix it also, but it is breaking change and will be done in other PR